paper.tex

```diff
@@ -1291,11 +1293,82 @@ \section*{Appendix}
+Recombination events happen at rate $\rho \nu / (m - 1)$, where $\rho \geq 0$ is a per-genome
+recombination rate and
+ \[
+ \nu = \sum_{x \in L}\left( \max_{(\ell, r, a) \in x}r
+     - \min_{(\ell, r, a) \in x}\ell - 1 \right)
+ \]
+ is the number of available ``links" surrounded by ancestral material.
+ At a recombination event we choose one of these links uniformly and break it, replacing the original lineage in $L$ with two new lineages containing the ancestral material to the left and right of the break point, respectively.
+
+Common ancestor events occur at rate $\binom{|L|}{2}$.
+In a common ancestor event, two uniformly sampled lineages have their segments merged into a single ancestor lineage, which is added to $L$.
+If the lineages have overlapping intervals of ancestry,
+say, $(\ell, r, a_1)$ and $(\ell, r, a_2)$, a
+\emph{coalescence} occurs. The result is a segment
+$(\ell, r, a_1 + a_2)$, and if $a_1 + a_2 < n$ it is included in the
+ancestor lineage. Otherwise, if $a_1 + a_2 = n$, we have found
+the most recent common ancestor of all samples in the interval$[\ell, r)$
+and do not need to simulate its history any further.
+Non-overlapping intervals from the two lineages are included
+ in the ancestor lineage without changes. Eventually,
+we find resultant lineages in which all segments have fully coalesced,
+and so the number of extant lineages gradually falls to zero.
+
+In the Griffiths formulation, each edge in the graph corresponds to an extant
+lineage and nodes are events in the process. The $n$ initial leaf nodes are
+sampling events. Common ancestor events occur at rate $\binom{|L|}{2}$.
+When a common ancestor event happens, two uniformly chosen lineages
+merge into a common ancestor lineage.
+Recombination events happen at rate $|L| \rho$. Here, we choose a lineage (i.e.\ edge) uniformly, and a
+breakpoint $0 < x < m$ uniformly on its genome. We terminate the edge at a
+node, record the breakpoint, and start two new edges from this node. The process
+then continues until there is only one lineage left (the Grand Most Recent
+Common Ancestor, GMRCA), which is guaranteed to
+happen in finite time because of the linear rate of coalescing vs.\ quadratic rate of branching.
+
+The state-space of the big ARG process is much simpler than that of the little ARG process,
+which greatly facilitates mathematical reasoning. This simplicity comes at a
+substantial cost, however, if we wish to use it as a practical means of
+simulating recombinant ancestries. 
+The number of events in the big ARG all the way back to the GMRCA
+is $O(e^\rho)$~\citep{griffiths1997ancestral}, whereas the number
+of events required to simulate the little ARG is
+$O(\rho^2)$~\citep{hein2004gene,baumdicker2021efficient}.
+This disparity arises because the majority of the events in the big ARG are 
+recombination events which occur outside of ancestral material,
+and this do not have any bearing
+on the ancestry of the initial sample.
```

Comment:
Maybe add another sentence here like "Because we don't keep track of the distribution of ancestral material during the process, we generate a vastly larger graph."

Reply:
Done, thanks.